### PR TITLE
Added two images; fixed ref to trial version

### PR DESCRIPTION
**Pull request by julieplummer20**

### CODE REVIEW

**Comment by thecodester** [comment]
I see no commits from today -- this version does not include errors we saw in QA?

File: tutorials/abap-environment-create-abapdoc/abap-environment-create-abapdoc.md [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+
+## Prerequisites
+- You have an entitlement to SAP Cloud Platform, ABAP Environment. For more information, see **Tutorial**: [Create Your First ABAP Console Application](abap-environment-console-application), steps 1-2
+- You have installed [ABAP Development Tools](https://tools.hana.ondemand.com/#abap), latest version
+- **Tutorial**: [Create an AMDP and Analyze Its Performance](abap-environment-amdp-profiling)
+
+## Details
+### You will learn  
+- How to maintain ABAPDoc comments
+- How to synchronize comments so they appear in the Outline View
```
Comment (thecodester) [inline tutorials/abap-environment-create-abapdoc/abap-environment-create-abapdoc.md:18]: Not clear why "View" is capitalized.

File: tutorials/abap-environment-create-abapdoc/abap-environment-create-abapdoc.md [hunk 2]
```diff
@@ -0,0 +1,131 @@
+---
+title: Create ABAPDoc Comments in Your Class in SAP Cloud Platform, ABAP Environment
+description: Learn how to maintain ABAPDoc documentation for your class
```
Comment (thecodester) [inline tutorials/abap-environment-create-abapdoc/abap-environment-create-abapdoc.md:3]: This description actually says less than the title!! Make title:

Create ABAPDoc Comments in Your Class

For the description, mention SAP Cloud Platform, ABAP Environment and make it more descriptive, maybe:

Learn how to maintain ABAPDoc documentation for your class in SAP Cloud Platform, ABAP Environment so your comments appear in the Outline view.

I assumed that adding comments and having them appear in the view were the coolest things the user would do, and therefore added them to the description.

**Comment by julieplummer20** [comment]
Sorry, something cropped up. Will do it tomorrow.

**Comment by julieplummer20** [comment]
Hi,
I tried to push but I got this error message:
Error: Command failed: git pull --quiet origin master
error: Your local changes to the following files would be overwritten by merge:
tutorials/abap-dev-enhance-cds-view/abap-dev-enhance-cds-view.md
Please commit your changes or stash them before you merge.
Aborting

Should I move abap-dev-enhance-cds-view to WIP locally to match the QA server?
What should I do?
Thanks Julie.

**Comment by thecodester** [comment]
@julieplummer20 I didn't actually see that you changed the description (at the top of the MD file with the description metadata). I will merge, but consider lengthening the official description.

**Pull request merged**